app/views/search/bento.html.erb

```diff
@@ -36,7 +36,5 @@
 
   <%= render partial: "trigger_search", locals: { target: 'books', id: 'books_content'} %>
 
-  <%= render partial: "trigger_search", locals: { target: 'whatnot', id: 'whatnot_content'} %>
```

Comment:
I must have missed something along the way here - I don't see a whatnot panel, either in production or in this branch - so I'm not sure what this changes?

This isn't a blocking comment, more a "what did I miss" question.